examples/pipelines/ccfraud/config.yaml

```diff
@@ -49,9 +49,9 @@ training_parameters:
   num_of_iterations: 3
   epochs: 3
   lr: 1e-3
-  batch_size: 1000
+  batch_size: 500
   # Property model_name defines model to be used for training and 
   # must be one of the models defined in the traininsilo/model.py 
   # file for the CCFRAUD component. Options include: 
   # SimpleLinear, SimpleLSTM, SimpleVAE
-  model_name: SimpleLSTM
+  model_name: SimpleLinear
```

Comment:
no problem for me but.... any reason you're changing the value here?

Reply:
I believe I changed it due to faster execution - just so I can test faster mainly on confidential compute as I was trying to execute it there as well